bitnami/mongodb/templates/common-scripts-cm.yaml

```diff
@@ -74,7 +74,7 @@ data:
     IP.0 = ${MY_POD_HOST_IP}
     IP.1 = 127.0.0.1
     EOL
-    index=1
+    index=2
```

Comment:
You need to also decrement the index for the DNS entries in line 84:
 https://github.com/bitnami/charts/blob/3f280328fb4281ca68d42965bcde7795094acd8c/bitnami/mongodb/templates/common-scripts-cm.yaml#L84

Reply:
yes, you are right...didn't recognize